### PR TITLE
Add "info button" to ClayManagementToolbar | Fixes #481

### LINKED ISSUE #481: Add "info button" to ClayManagementToolbar
New feature required by UX: https://docs.google.com/document/d/1y67UAnbUzL5WLAkPMQbkqgGw3_5e7-JMEPVYdC48qE0/edit?ts=5a692baa#

### PR DESCRIPTION
**Pull request by matuzalemsteles**

### CODE REVIEW

File: packages/clay-management-toolbar/src/ClayManagementToolbar.js [hunk 1]
```diff
@@ -220,6 +229,15 @@ ClayManagementToolbar.STATE = {
...
 	id: Config.string(),
 
+	/**
+	 * Flag to indicate if the Info button is needed.
+	 * @instance
+	 * @memberof ClayManagementToolbar
+	 * @type {?bool}
+	 * @default false
+	 */
+	infoButton: Config.bool().value(false),
```
Comment (carloslancha) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.js]: For consistency with the rest of the params I think this should be named `showInfoButton`

File: packages/clay-management-toolbar/src/ClayManagementToolbar.js [hunk 2]
```diff
@@ -220,6 +229,15 @@ ClayManagementToolbar.STATE = {
 	 */
 	id: Config.string(),
 
+	/**
+	 * Flag to indicate if the Info button is needed.
```
Comment (carloslancha) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.js]: To be more precise: `Flag to indicate if the Info button should be shown or not.`

File: packages/clay-management-toolbar/src/ClayManagementToolbar.soy [hunk 3]
```diff
@@ -52,6 +53,7 @@
 	{@param? handleViewTypeClicked_: any}
 	{@param? hideFiltersDoneButton: bool}
 	{@param? id: string}
+	{@param? infoButton: bool}
```
Comment (carloslancha) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: For consistency with the rest of the params I think this should be named `showInfoButton`

File: packages/clay-management-toolbar/src/ClayManagementToolbar.soy [hunk 4]
```diff
@@ -91,13 +93,15 @@
...
 			{param handleFilterDoneButtonClick_: $handleFilterDoneButtonClick_ /}
+			{param handleInfoButtonClicked_: $handleInfoButtonClicked_ /}
 			{param handleOpenMobileSearchClick_: $handleOpenMobileSearchClick_ /}
 			{param handleSearchSearchClick_: $handleSearchSearchClick_ /}
 			{param handleSelectPageCheckboxChanged_: $handleSelectPageCheckboxChanged_ /}
 			{param handleSortingButtonClicked_: $handleSortingButtonClicked_ /}
 			{param handleViewTypeClicked_: $handleViewTypeClicked_ /}
 			{param hideFiltersDoneButton: $hideFiltersDoneButton /}
 			{param id: $id /}
+			{param infoButton: $infoButton /}
```
Comment (carloslancha) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: For consistency with the rest of the params I think this should be named `showInfoButton`

File: packages/clay-management-toolbar/src/ClayManagementToolbar.soy [hunk 5]
```diff
@@ -248,13 +252,15 @@
...
 	{@param? handleFilterDoneButtonClick_: any}
+	{@param? handleInfoButtonClicked_: any}
 	{@param? handleOpenMobileSearchClick_: any}
 	{@param? handleSearchSearchClick_: any}
 	{@param? handleSelectPageCheckboxChanged_: any}
 	{@param? handleSortingButtonClicked_: any}
 	{@param? handleViewTypeClicked_: any}
 	{@param? hideFiltersDoneButton: bool}
 	{@param? id: string}
+	{@param? infoButton: bool}
```
Comment (carloslancha) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: For consistency with the rest of the params I think this should be named `showInfoButton`

File: packages/clay-management-toolbar/src/ClayManagementToolbar.soy [hunk 6]
```diff
@@ -367,6 +373,19 @@
 					{/call}
 				</li>
 
+				{if $infoButton}
```
Comment (carloslancha) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: For consistency with the rest of the params I think this should be named `showInfoButton`

**Comment by matuzalemsteles** [comment]
hey @carloslancha, Yeah, I talked to Patrick about this, I think we can go with that to provide the API. Thank you for your thoughts!

File: packages/clay-management-toolbar/src/ClayManagementToolbar.js (hunk 1 above)
Comment (matuzalemsteles) [reply packages/clay-management-toolbar/src/ClayManagementToolbar.js]: Oh yeah, it looks so much better now. Thanks!

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Resent https://github.com/liferay/clay/pull/517

**Pull request closed without merging**